### PR TITLE
MS Windows - Build using VCPKG

### PR DESCRIPTION
**Pull request by vcebollada**
* Uses the third-party libraries built by VCPKG.

Signed-off-by: Victor Cebollada <v.cebollada@samsung.com>

### CODE REVIEW

File: vcpkg-script/buiuld-deps.sh [hunk 1]
```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
```
Comment (adeelkazmi) [inline vcpkg-script/buiuld-deps.sh]: file name is incorrect (extra u).

File: README.md [hunk 2]
```diff
@@ -47,4 +47,11 @@ Run the .bat files to config the enviorment
     windows-dependencies\setenv.bat
 
 Step5:
+Install vcpkg to build all the third-party dependecies: go to vcpkg-scripts, read the Readme.md file for more instructions open a bash shell for MS Windows and execute the script to install vcpkg.
```
Comment (adeelkazmi) [inline README.md]: Folder name is vcpkg-script, either change the folder name or correct here.

Not sure what you mean by bash shell, is it the command line you get on windows or windows powershell or even WSL.

File: vcpkg-script/Readme.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+###Instructions to build the third-party dependencies
```
Comment (adeelkazmi) [inline vcpkg-script/Readme.md]: Formatting for this file is incorrect. Have a look here:
https://github.com/vcebollada/windows-dependencies/tree/vcpkg/vcpkg-script

File: vcpkg-script/Readme.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+The git bash app will be used to run the script.
+- Set a proxy ip might be needed.
+The default http proxy ip is set to the one in SRUK
+Use the following options as a parameter for the script to set a different proxy ip:
+-p | -httpProxy | --httpProxy To set the http proxy ip. It sets as well the https one
+[-s | -httpsProxy | --httpsProxy] Optional. To set the https proxy ip if it's different than the http one.
+[-n] Optional. Doesn't set any proxy.
+
+2.Create a folder where to install VCPKG. Better if this forder is outside of the watch3d folder. i.e C:\Tools\VCPKG
+3.Copy the Watch3D_Build_Deps.sh file and all the patch files to the newly created folder.
```
Comment (adeelkazmi) [inline vcpkg-script/Readme.md]: Watch?

Actually are these instructions even valid?

**Comment by adeelkazmi** [comment]
Getting this error when I run the vcppkg shell script:

patch: **** Can't open patch file '../[VCPKG]_0001_Fix_proxy_access.patch' : No such file or directory
patch: **** Can't open patch file '../[VCPKG]_0002_Fix_Windows_toolchain_bigobj.patch' : No such file or directory
patch: **** Can't open patch file '../[VCPKG-angle]_0001_Apply_Fix_windows_build_error_cannot_find_definition_far.patch' : No such file or directory
patch: **** Can't open patch file '../[VCPKG-angle]_0002_Fix_angle_header_files.patch' : No such file or directory
patch: **** Can't open patch file '../[VCPKG-angle]_0003_Apply_Fix_glInvalidateFramebuffer_crash.patch' : No such file or directory
patch: **** Can't open patch file '../[VCPKG-fontconfig]_0001_Apply_Source_compile_fix.patch' : No such file or directory
patch: **** Can't open patch file '../[VCPKG-getopt]_0001_Apply_Fix_extern_c.patch' : No such file or directory
patch: **** Can't open patch file '../[VCPKG-libjpeg-turbo]_0001_Apply_Fix_fill_jpeg_buffer_cb.patch' : No such file or directory
patch: **** Can't open patch file '../[VCPKG-pthreads]_0001_Apply_Fix_define_timespec.patch' : No such file or directory

**Review by adeelkazmi — CHANGES_REQUESTED** [review]
See previous comments

File: vcpkg-script/buiuld-deps.sh (hunk 1 above)
Comment (vcebollada) [reply vcpkg-script/buiuld-deps.sh]: done

**Pull request merged**